front/components/assistant/conversation/input_bar/InputBarContainer.tsx

```diff
@@ -149,8 +149,14 @@ const InputBarContainer = ({
         }));
       });
 
-      if (nodesWithViews.length > 0) {
-        const sortedNodes = nodesWithViews.sort(
+      const nodes = nodesWithViews.filter(
+        (node) =>
+          isNodeCandidate(nodeOrUrlCandidate) ||
+          node.sourceUrl === nodeOrUrlCandidate?.url
```

Comment:
Let's maybe add a comment here, because the day we do some more fancy urls matching (other than exact match), this will be breaking right?